### PR TITLE
`path/globRoot` enhancements

### PR DESCRIPTION
**Pull request by pskfyi**
Extracted from #53

### CODE REVIEW

File: path/globRoot.ts [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+ * globRoot("/foo/bar/*.ts") // "/foo/bar/"
+ * globRoot("C:\\\\foo\\bar\\*.ts") // "C:\\\\foo\\bar\\"
+ * globRoot("foo/bar") // "foo/bar"
+ */
 export function globRoot(glob: string) {
   let root = "";
 
-  const parts = glob.split(sep);
-  const lastIndex = String(parts.length - 1);
+  const parts = glob.split(/([\/\\])/);
```
Comment (pskfyi) [inline path/globRoot.ts:13]: This allows the test suite to pass in all 3 OS's.

File: path/globRoot.test.ts [hunk 2]
```diff
@@ -1,16 +1,24 @@
-import { join, resolve, sep } from "../_deps/path.ts";
 import { assertEquals, describe, it } from "../_deps/testing.ts";
 import { globRoot } from "./globRoot.ts";
 
-const PROJECT_ROOT_DIR = resolve(".");
```
Comment (pskfyi) [inline path/globRoot.test.ts:5]: This shouldn't have been here, which prompted me to rethink this entire file.

File: _deps/path.ts [hunk 3]
```diff
@@ -6,6 +6,5 @@ export {
   isGlob,
   join,
   resolve,
-  sep,
```
Comment (pskfyi) [inline _deps/path.ts:9]: `path/globRoot.ts` is the only place this was used 🎉

**Review by AustinArey — APPROVED** [review]
Elegant solution!

**Pull request merged**